rspec/execute_spec.py

```diff
@@ -57,7 +57,7 @@ def _command_hash(self):
     add_to_path = self.context.from_settings("rspec_add_to_path", "")
     append_path = "export PATH={0}:$PATH;".format(add_to_path) if add_to_path else ""
 
-    command = "({append_path} cd {project_root} && {rspec_command} {target})".format(
+    command = "({append_path} cd '{project_root}' && {rspec_command} {target})".format(
```

Comment:
I think this would fail in the unlikely event project root contains `'`.

There's a function we can use to escape it: https://docs.python.org/dev/library/shlex.html#shlex.quote

So I suggest we do `project_root = shlex.quote(self.context.project_root()),` instead.

Reply:
@olivierlacan ^^